### PR TITLE
replace support.crowdin.com domain to developer.crowdin.com domain ev…

### PR DESCRIPTION
**Pull request by kmartin62**
…erywhere in the code

### CODE REVIEW

**Comment by kmartin62** [comment]
Hey @andrii-bodnar, this is supposed to fix #99. Please review it and do let me know if something needs to be changed.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[<p align="center"><img src="https://support.crowdin.com/assets/logos/crowdin-dark-symbol.png" data-canonical-src="https://support.crowdin.com/assets/logos/crowdin-dark-symbol.png" width="200" height="200" align="center"/></p>](https://crowdin.com)
+[<p align="center"><img src="https://developer.crowdin.com/assets/logos/crowdin-dark-symbol.png" data-canonical-src="https://developer.crowdin.com/assets/logos/crowdin-dark-symbol.png" width="200" height="200" align="center"/></p>](https://crowdin.com)
```
Comment (andrii-bodnar) [inline README.md:1]: The logo links shouldn't be changed

File: src/CrowdinApiClient/Api/BranchApi.php [hunk 2]
```diff
@@ -16,8 +16,8 @@ class BranchApi extends AbstractApi
 {
     /**
      * List Branches
-     * @link https://support.crowdin.com/api/v2/#operation/api.projects.branches.getMany API Documentation
-     * @link https://support.crowdin.com/enterprise/api/#operation/api.projects.branches.getMany API Documentation Enterprise
+     * @link https://developer.crowdin.com/api/v2/#operation/api.projects.branches.getMany API Documentation
+     * @link https://developer.crowdin.com/enterprise/api/#operation/api.projects.branches.getMany API Documentation Enterprise
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/BranchApi.php:20]: This link is broken

For Crowdin Enterprise it should be like this:

`https://support.crowdin.com/enterprise/api/*` -> `https://developer.crowdin.com/enterprise/api/v2/*`

**Comment by andrii-bodnar** [comment]
Hi @kmartin62, thanks a lot for your contribution!

Please check the comments I've left

**Comment by kmartin62** [comment]
> Hi @kmartin62, thanks a lot for your contribution!
> 
> Please check the comments I've left

Thanks for the review! If I understood you well, I changed it to `enterprise/api/v2`. Hopefully this time will be okay, let me know if I'd need to change something again.

**Review by andrii-bodnar — APPROVED** [review]

**Comment by kmartin62** [comment]
> @kmartin62 looks good to me, thank you!

You're welcome! I'm doing this as part of Hacktoberfest, it's my first time contributing to OS projects and it was pleasure to work on this.

**Comment by andrii-bodnar** [comment]
@kmartin62 congratulations on your first contribution 🎉 

We are happy to accept contributions, so feel free to submit other pull requests, there are a [few more repositories](https://github.com/search?q=topic%3Ahacktoberfest+org%3Acrowdin+fork%3Atrue&type=repositories) in this organization with the `hacktoberfest` labeled issues 🙂

**Pull request merged**